ui/src/sources/containers/ManageSources.tsx

```diff
@@ -82,33 +78,32 @@ class ManageSources extends Component {
               handleDeleteSource={this.handleDeleteSource}
               setActiveKapacitor={this.handleSetActiveKapacitor}
             />
-            <p className="version-number">Chronograf Version: {V_NUMBER}</p>
+            <p className="version-number">
+              Chronograf Version: {V_NUMBER}
+            </p>
           </div>
         </FancyScrollbar>
       </div>
     )
   }
-}
 
-const {array, func, shape, string} = PropTypes
-
-ManageSources.propTypes = {
-  source: shape({
-    id: string.isRequired,
-    links: shape({
-      proxy: string.isRequired,
-      self: string.isRequired,
-    }),
-  }),
-  sources: array,
-  notify: func.isRequired,
-  removeAndLoadSources: func.isRequired,
-  fetchKapacitors: func.isRequired,
-  setActiveKapacitor: func.isRequired,
-  deleteKapacitor: func.isRequired,
+  private handleDeleteSource = (source: Source) => () => {
```

Comment:
could you add return types to both functions in this signature, even if `void`? that will prevent having to revisit this once we flip the switch on `no-implicit-any`